app/controllers/resources_controller.rb

```diff
@@ -5,7 +5,11 @@ class ResourcesController < RestController
   include AssumedRole
   
   def index
-    options = params.slice(:account, :kind, :limit, :offset, :search).symbolize_keys
+    # Rails 5 requires parameters to be explicitly permitted before converting 
+    # to Hash.  See: https://stackoverflow.com/a/46029524
+    allowed_params = %i(account kind limit offset search)
```

Comment:
Should this be a class-level constant?

Reply:
It's only relevant to this method.  Technically you're right to see this as a code smell but it's the kind of bad designed encouraged by Rails framework.  I'd personally rather retain locality than create `ALLOWED_INDEX_PARAMS` or do a larger refactoring.